sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileSystems.java

```diff
@@ -69,7 +69,7 @@
 @Experimental(Kind.FILESYSTEM)
 public class FileSystems {
 
-  public static final String DEFAULT_SCHEME = "default";
```

Comment:
Update the class comment to better state what FileSystems is about specifically stating that if a filesystem is not found, `FileSystems` will attempt to use the local file system to resolve the path.